airgun/views/acs.py

```diff
@@ -0,0 +1,447 @@
+            ssl_client_key = Text('//dd[@aria-label="sslClientKey_value"]')
+            username = Text('//dd[@aria-label="username_value"]')
+            password = Text('//dd[@aria-label="password_value"]')
+
+    @View.nested
+    class products_stack_item(View, AcsStackItem):
+        """
+        Class representing the Products stack item in the ACS drawer.
+        Present only if ACS is of type 'Simplified'.
+        """
+
+        ROOT = (
+            '//div[normalize-space(.)="Products" and contains(@class, "pf-c-expandable-section")]'
+        )
+
+        title = OUIAText('expandable-products-text')
+        edit_products = Button(
+            locator='//button[contains(@aria-label, "edit-products-pencil-edit")]'
+        )
+
+        @View.nested
+        class products_stack_content(View):
+            """Class representing content of the Products stack item."""
+
+            ROOT = '//div[@id="showProducts"]'
+
+            products_list = ItemsList(locator='.//ul[contains(@class, "pf-c-list")]')
+
+    def read(self):
+        """Reads the ACS drawer and returns its content as a dictionary."""
+
+        keys = ['details', 'capsules', 'url_and_subpaths', 'credentials', 'products']
+        result = dict.fromkeys(keys)
+        # Sleep ensures that all stack items are loaded and test does not fail
+        time.sleep(3)
+        # Expand and read each stack item
+        for key in result:
+            if key == 'details':
+                self.details_stack_item.expand()
+                result[key] = self.details_stack_item.read()
+                result['details']['last_refresh'] = self.last_refresh.text
+            elif key == 'capsules':
+                self.capsules_stack_item.expand()
+                result[key] = self.capsules_stack_item.read()
+            elif key == 'url_and_subpaths':
+                if self.url_and_subpaths_stack_item.is_displayed:
+                    self.url_and_subpaths_stack_item.expand()
+                    result[key] = self.url_and_subpaths_stack_item.read()
+            elif key == 'credentials':
+                if self.credentials_stack_item.is_displayed:
+                    self.credentials_stack_item.expand()
+                    result[key] = self.credentials_stack_item.read()
+            elif key == 'products':
+                if self.products_stack_item.is_displayed:
+                    self.products_stack_item.expand()
+                    result[key] = self.products_stack_item.read()
+
+        # Remove None values from the result dictionary and return it
+        result = {k: v for k, v in result.items() if v is not None}
+        return result
+
+
+class AlternateContentSourcesView(BaseLoggedInView):
+    """Class that describes view of the Alternate Content Sources page."""
+
+    title = Text('//h1[contains(., "Alternate Content Sources")]')
+    error_message = Text('//div[contains(@aria-label, "Danger Alert")]')
+
+    @View.nested
```

Comment:
I do not think, this is a `nested view`. looking at the UI I feel it is another All entity page view.